man/gbz80.7

```diff
@@ -758,10 +788,11 @@ Flags: None affected.
 .Ss JP HL
 Jump to address in
 .Sy HL ;
-effectively, load
-.Sy PC
-with value in register
-.Sy HL .
+effectively, store
```

Comment:
I wonder if "copy" wouldn't be a good idea, actually? I remember being confused when I started about what would happen to the register the value is loaded *from*.

Reply:
Sure, "copy *assignee* into *destination*" would be a good convention too.